api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
`make generate` automatically seems to be adding this now. If we remove this line, the ci fails. Not sure if there is a workaround for this.

Reply:
It's actually the `go fmt` command that's doing this as of golang 1.17: https://stackoverflow.com/questions/10646531/golang-conditional-compilation/67937234#67937234

If we're moving to golang 1.18, we shouldn't need to work around this.

Reply:
I find that this happens when I use vscode
whenever I need to get rid of this I edit the file in the native fedora editor and that sticks